lib/inventory_refresh/inventory_collection.rb

```diff
@@ -232,13 +226,13 @@ def unique_index_keys
 
     # @return [Array<ActiveRecord::ConnectionAdapters::IndexDefinition>] array of all unique indexes known to model
     def unique_indexes
-      @unique_indexes_cache if @unique_indexes_cache
+      @unique_indexes_cache
```

Comment:
Was this a bug?  Right now it's a no-op

Reply:
yeah I think so, I'm guessing this was supposed to be `return @unique_indexes_cache if @unique_indexes_cache`